src/core/instance/instance.hpp

```diff
@@ -1030,6 +1030,11 @@ template <> inline Utils::Otns &Instance::Get(void) { return mOtns; }
 template <> inline BorderRouter::RoutingManager &Instance::Get(void) { return mRoutingManager; }
 
 template <> inline BorderRouter::InfraIf &Instance::Get(void) { return mRoutingManager.mInfraIf; }
+
+template <> inline BorderRouter::RoutingManager::PdPrefixManager &Instance::Get(void)
+{
+    return mRoutingManager.mPdPrefixManager;
+}
```

Comment:
Remove this (we should not need this)?

Also `PdPrefixManager` is a `private` class within `RoutingManager` class so should not be visible/known by other modules.